lib/phx_component_helpers.ex

```diff
@@ -128,7 +128,7 @@ defmodule PhxComponentHelpers do
   def validate_required_attributes(assigns, nil), do: assigns
 
   def validate_required_attributes(assigns, required) do
-    missing = for attr <- required, !Map.has_key?(assigns, attr), do: attr
+    missing = for attr <- required, !Map.get(assigns, attr), do: attr
```

Comment:
And what if we have a `assigns=%{foo: false}` foo is `required`?

Reply:
Thank you, fixed in v1.0.3 :)